core/src/solve.jl

```diff
@@ -79,17 +84,19 @@ function set_current_basin_properties!(
     t::Number,
 )::Nothing
     (; basin) = p
+    (; current_properties, cumulative_precipitation, cumulative_drainage, vertical_flux) =
+        basin
     (;
         current_storage,
+        current_low_storage_factor,
         current_level,
         current_area,
         current_cumulative_precipitation,
         current_cumulative_drainage,
-        cumulative_precipitation,
-        cumulative_drainage,
-        vertical_flux,
-    ) = basin
+    ) = current_properties
+
     current_storage = current_storage[parent(du)]
+    current_low_storage_factor = current_low_storage_factor[parent(du)]
```

Comment:
This parameter should have a comment (in the PR description, or at the paramter.jl level) as it's new functionality.